process/block/preprocess/sovereignSmartContractResultPreProcessorFactory.go

```diff
@@ -0,0 +1,45 @@
+package preprocess
+
+import (
+	"github.com/multiversx/mx-chain-core-go/core/check"
+	"github.com/multiversx/mx-chain-go/process"
+)
+
+// SmartContractResultPreProcessorCreator defines the interface of a smart contract result pre-processor creator
+type SmartContractResultPreProcessorCreator interface {
```

Comment:
Could be this moved in process/interface.go or process/block/interface.go or process/block/preprocess/interface.go ?

Reply:
moved to process/block/preprocess/interface.go